.gitignore

```diff
@@ -5,3 +5,4 @@
 *~
 *.iml
 composer.phar
+app-prod.yaml
```

Comment:
is there a precedent for this? I've never seen this convention. Aren't prod credentials just stored using `gcloud auth login` and `gcloud config set project`?